src/vhost_user/master.rs

```diff
@@ -360,7 +360,9 @@ impl VhostUserMaster for Master {
         // "Master payload: virtio device config space"
         // But what content should the payload contains for a get_config() request?
         // So current implementation doesn't conform to the spec.
-        let hdr = node.send_request_with_body(MasterReq::GET_CONFIG, &body, None)?;
+        let mut buf: Vec<u8> = Vec::with_capacity(body.size as usize);
+        buf.resize(body.size as usize, 0);
```

Comment:
this may be simplified as 
let buf = vec![0u8; body.size as usize];

Reply:
done, thanks